source/nanoFramework.Hardware.Esp32/TouchPad/TouchPad.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace nanoFramework.Hardware.Esp32.TouchPad
+{
+/// <summary>
+/// TouchPad sensor using interrupts
+/// </summary>
+  public sealed class TouchPad : TouchPadBase
+  {
+    public delegate void TouchPadValueChangedEventHandler(
+        Object sender);
+		//TouchPadValueChangedEventArgs e);
```

Comment:
It makes sense to have a specific event for touchpads. This has to be added in the events class lib first.

Reply:
Shall I comment out those lines, delete it or leave it as it is?

Reply:
First need to add this event to Events class lib.
It's OK if you add it there.

Submit a PR in lib-nanoFramework.Runtime.Events. Basically you need to add a new enumerable to EventCategory.